### DISCUSSION TITLE
Giveaway email: phishing or legitimate?

### DISCUSSION THREAD

**Discussion by perey** [community/community#23923] [New to GitHub]
<p>I’ve received an email claiming to be from GitHub, offering a free gift if I just give them some personal details (name, physical address, etc.). No catch, no terms and conditions, no nothing. Looks like a classic phishing scam, right?</p>
<p><img src="https://user-images.githubusercontent.com/1608621/181080038-0d213018-f8c2-4131-8506-1ddfccf4fd15.png" alt="github-sus-01" data-base62-sha1="2lu50zlXFu0iV1MMDA6wE0UeAFJ" width="526" height="135"></p>
<p>The thing is, everything points to it really being from GitHub. I’m not up on the state of the art in how emails are signed and senders verified, but here’s what Gmail reports:</p>
<ul>
<li>mailed-by: <a href="http://sgmail.github.com" rel="noopener nofollow ugc">sgmail.github.com</a>
</li>
<li>Signed by: <a href="http://github.com" rel="noopener nofollow ugc">github.com</a>
</li>
<li>Security: Standard encryption (TLS)</li>
</ul>
<p>I’ve looked over the GitHub help for some way to report suspicious emails, but all I can find is how to report abusive or spamming <em>users</em>. So I’ve come here hoping for help. Can anyone confirm or refute the legitimacy of this email?</p>

**Comment by tuves** [comment, marked as answer]
<p>Hi <a class="mention" href="/perey">@perey</a>,</p>
<p>After some digging–I found that this email is legitimate and from GitHub.</p>

**Comment by perey** [comment]
<p>Thanks for that. Is there any way that you know of for my feedback to get to the people behind it, suggesting they make sure there’s an easier way to confirm this sort of thing is for real?</p>

**Comment by tuves** [comment]
<p>Hi <a class="mention" href="/perey">@perey</a>–please do feel free to share your feedback here. I’ll make sure it gets back to the appropriate team(s). I’ve also shared this thread with them for visibility. :smiley:</p>

**Comment by perey** [comment]
<p>The public are told all the time to beware of giving out information—even to (what appears to be) trusted sites, if they aren’t absolutely sure it’s genuine. Asking via email, and collecting information on a third-party website (Google Forms), are warning signs. So when a <strong>legitimate</strong> survey comes along, it needs to avoid:</p>
<ul>
<li>Raising incorrect suspicions (reduces number of replies received, wastes support/community time in answering threads like this one).</li>
<li>Undermining security messages (<em>this</em> one was legit, so people are more likely to hand out information next time).</li>
</ul>
<p>What I think anyone running a survey/giveaway needs to do is to have a source of information about it, separate from the request and verifiably from the genuine organisation (so, on the GitHub website).</p>
<p>At a minimum, there should be some statement like: yes, we sent this email, and yes, we want you to go fill out this Google Form. I expect, and I think most people expect, to see terms and conditions as well. Ideally there’d also be some confirmation: your survey has (or hasn’t) been received and we’re including (or not including) you in the giveaway.</p>
<p>I recognise that the survey that prompted this was meant for a limited number of people, and secrecy and transparency can be hard to balance (how do you publish terms and conditions for a private offer without making it public?). But surely there are best practices out there for this already.</p>
